### PR TITLE
Pr logrotate

### PR DESCRIPTION
**Pull request by DirectedSoul1**
# Description

This PR uses `logrotate` using cron to compress all larger files > 5MiB from `/var/log/messages` 

## Type of change

Please select the appropriate options:

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have performed a self-review of my own code
- [x] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [x] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by iranzo** [comment]
I think that changelog should be left out of the PR, as it is managed entirely by the release GHA

File: hack/deploy-hub-local/ansible/setup.yaml [hunk 1]
```diff
@@ -79,3 +79,44 @@
...
+
+    - name: Add Slack notifications
+      community.general.slack:
+        token: #Add-your-slack-token
+        msg: '{{ inventory_hostname }} completed'
+        channel: #Add-your-slack-channel
+      delegate_to: localhost
+      tags: [ 00-slack ]
+
+    - name: Generate an OpenSSH keypair with the default values (4096 bits, rsa)
```
Comment (fbac) [inline hack/deploy-hub-local/ansible/setup.yaml:91]: I would leave all the ssh related actions inside the ssh subdirectory, and with the same tag used currently, so we can force/skip it whenever needed.
Also maybe add some vars? For example forcing a certain key type, key length, etc.

File: hack/deploy-hub-local/ansible/setup.yaml [hunk 2]
```diff
@@ -79,3 +79,44 @@
...
+      community.crypto.openssh_keypair:
+        path: /tmp/id_ssh_rsa
+        backend: auto
+        force: yes
+      tags: [ 01-ssh ]
+
+
+    - name: Install logrotate
```
Comment (fbac) [inline hack/deploy-hub-local/ansible/setup.yaml:99]: Same as with sshd actions, for logrotate maybe creating a subdirectory for all its specific actions its more convenient, and all of them using a common tag such as [configure-logrotate] using the same name convention.

File: hack/deploy-hub-local/ansible/setup.yaml [hunk 3]
```diff
@@ -79,3 +79,44 @@
       service:
         name: sshd
         state: restarted
+
+    - name: Add Slack notifications
```
Comment (fbac) [inline hack/deploy-hub-local/ansible/setup.yaml:83]: Not sure about this task regarding recent discussions, probably we'll have to leave it out of the playbooks until we figure the way we want to be notified about the server statuses.

**Review by fbac — CHANGES_REQUESTED** [review]
Hey shashank o/

I added a couple comments, also please take a look at setup.yaml, as in the finalizing task you'll have to include the added labels, so it's executed whatever the label you're running the playbook with.

```
    - name: Finalize playbook
      include_tasks: tasks/finalizers/99-finalizers.yaml
      tags: [ 00-base-os, 00-base-network, 01-packages, 
            01-configure-dnf, 02-bin-utils, 
            00-common-services, configure-sshd,
            configure-tuned, configure-chronyd,
            configure-logrotate, configure-libvirtd ]
```

**Comment by fbac** [comment]
I'll be closing this PR as I'm assigning to myself this task to liberate shegde. The work will be resumed in some time.

**Pull request closed without merging**